crates/viewer/re_renderer/shader/rectangle_fs.wgsl

```diff
@@ -1,11 +1,6 @@
 #import <./colormap.wgsl>
 #import <./rectangle.wgsl>
 #import <./utils/srgb.wgsl>
-#import <./decodings.wgsl>
-
-// WARNING! Adding anything else to this shader is very likely to push us over a size threshold that
-// causes the failure reported in: https://github.com/rerun-io/rerun/issues/3931
-// Make sure any changes are tested in Chrome on Linux using the Intel Mesa driver.
```

Comment:
the _expectation_ is that this is no longer the case now.
Is it true though? @jleibs :)